themes/scss/widgets/text-elements.scss

```diff
@@ -6,10 +6,9 @@
 
 .Widget-textBigger { // 16 size
   @include text-overflow();
+  color: #2E3C43;
```

Comment:
Don't add colors to the element classes, let's add them to the light theme, please.